File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.java
class Solution {
    public int averageValue(int[] nums) {
        int sum=0;
        int i=0;
        for(int n:nums){
            if(n%6==0){
                sum+=n;
                i++;
            }
        }
        return i>0?sum/i:sum;
    }
}

File: 2455-average-value-of-even-numbers-that-are-divisible-by-three/2455-average-value-of-even-numbers-that-are-divisible-by-three.py
class Solution(object):
    def averageValue(self, nums):
        sum=0
        j=0
        for i in nums:
            if i%6==0:
                sum=sum+i
                j=j+1
        avg=sum
        if j>0:
            avg=sum/j
        
        return avg

        